communication/slack-config/restrictions.yaml

```diff
@@ -10,6 +10,14 @@ restrictions:
       - ".*"
   - path: template.yaml
     template: true
+  - path: "sig-architecture/*.yaml"
+    channels:
+      - "^sig-architecture$"
+      - "^enhancements$"
+      - "^k8s-code-organization$"
+      - "^k8s-conformance$"
```

Comment:
Also "^prod-readiness$". So with this allow us to do "@here" etc?

Reply:
Nope, `@here` is restricted to admins. Channel level moderation / capabilities might be an eventual thing, but this really just delegates ownership so you know which channels belong to sig-arch. If there was a naming convention like `sig-arch-<foo>` we could use that pattern and allow sig-arch-leads to create any channels for themselves as long as they followed that pattern.

Reply:
Agreed, but let's punt on naming convention until after this merges, so I can close the books on the SIG PM stuff.